django_react_proj/users-fe/src/components/RealTime/RealTime.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import moment from 'moment'
+import styles from "./RealTime.module.css"
+
+const useCurrentCallback = (callback) => {
+    const reference = React.useRef();
+    reference.current = callback;
+    return (...args) => {
+      return reference.current?.(...args);
+    };
+  };
+  
+const Time = () => {
+    const [time, setTime] = React.useState(0);
+    const currentCallback = useCurrentCallback(() => {
```

Comment:
I feel like this abstraction is unnecessary for what you want to achieve, can you explain what is it for and what this component intends to do (I might be missing something)?

Reply:
Now i looked on the code and i saw it could be done in more easier way without useCurrentCallBack